lib/History/History.js

```diff
@@ -89,6 +96,12 @@ var History = module.exports = kind.singleton({
 		if (this.enableBackHistoryAPI) {
 			this._initHistoryState();
 		}
+
+		this._popStateHandler = this.bindSafely(function () {
```

Comment:
Not a big deal but there's no need to wrap `popStateHandler` with another function. `this.bindSafely(this.popStateHandler)` should do the trick, right?